.github/workflows/main.yml

```diff
@@ -0,0 +1,70 @@
+name: Mac OS build
+
+on:
+  push
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install dependencies
+      run: brew install gtk+3 gtkmm3 gtk-mac-integration adwaita-icon-theme libsigc++ little-cms2 libiptcdata fftw lensfun llvm expat pkgconfig libomp shared-mime-info
+    - name: patch libiconv
+      run: |
+        mkdir libiconv && cd libiconv
+        wget https://ftp.gnu.org/pub/gnu/libiconv/libiconv-1.16.tar.gz
+        tar xf libiconv-1.16.tar.gz
+        cd libiconv-1.16
+        patch -p1 < "${GITHUB_WORKSPACE}/tools/osx/libiconv_1.16_rt.patch"
+        mkdir build && cd build
+        destDir="$(pwd)"
+        ../configure --prefix=/opt/local --disable-static 'CFLAGS=-arch x86_64 -mmacosx-version-min=10.9' 'LDFLAGS=-arch x86_64 -mmacosx-version-min=10.9' CXXFLAGS="-arch x86_64 -mmacosx-version-min=10.9"
+        make --jobs 2
+        make DESTDIR="${destDir}" install
+        sudo mv opt/local /usr/local/opt/libiconv
+    - name: print /usr/local/lib contents
+      run: ls -al /usr/local/lib
+    - name: cmake
+      env:
+        CMAKE_CXX_STANDARD: 11
+        # clutching at the straw https://stackoverflow.com/questions/17912092/gtk-installed-with-brew-is-not-found-when-i-try-to-compile-other-applications with /opt/X11
+        PKG_CONFIG_PATH: /usr/local/opt/libffi/lib/pkgconfig:/usr/local/opt/expat/lib/pkgconfig:/opt/X11/lib/pkgconfig
+        RAW_THERAPEE_MAJOR: '5'
+        RAW_THERAPEE_MINOR: '7'
+        C_FLAGS: -Xpreprocessor -fopenmp /usr/local/lib/libomp.dylib -I/usr/local/include -I/usr/local/opt/gdk-pixbuf/include -I/usr/local/opt/libiconv/include -I/usr/local/opt/libxml2/include -I/usr/local/opt/expat/include -I/usr/local/opt/llvm/include
+      run: |
+        REF=${GITHUB_REF##*/}
```

Comment:
@Krijger when debugging scripts in the future, it's nice when the code comments show examples of what the input and output are meant to be. What are GITHUB_REF and REF?

Reply:
added requested info